src/adapter/search-request-adapter/search-params-adapter.ts

```diff
@@ -63,6 +63,22 @@ export function adaptSearchParams(
     '*',
   ]
 
+  // Highlight pre tag
+  const highlightPreTag = searchContext?.highlightPreTag
+  if (highlightPreTag) {
+    meiliSearchParams.highlightPreTag = highlightPreTag
+  } else {
+    meiliSearchParams.highlightPreTag = '__ais-highlight__'
```

Comment:
`__ais-highlight__` is the default highlight of instant-search?

Reply:
Absolutely! It is actually a bad idea to change this in instant-meilisearch as the tags are not rendered as HTML. 

<img width="261" alt="Screenshot 2022-04-25 at 12 23 23" src="https://user-images.githubusercontent.com/33010418/165071068-d7777292-96ef-471d-8efe-40997f59d78b.png">

So the user should stick with the default value. 
I added it because the user might want to add other things than tags and also to be consistent with our search parameters.